rca/enquire_to_study/views.py

```diff
@@ -0,0 +1,55 @@
+from django.utils.decorators import method_decorator
+from django.views.decorators.csrf import csrf_exempt
+from django.views.generic import FormView, TemplateView
+
+from .forms import EnquireToStudyForm
+
+
+class EnquireToStudyFormView(FormView):
+    """Cutsom form with integrations to Mailchimp and QS
+
+    TODO:
+        - The form will need to generate a submission object, so we can
+            create a dataclip to send to google as a cv export.
+        - Mailchimp Integration
+        - QS Integration
+    """
+
+    form_class = EnquireToStudyForm
+    success_url = "/enquire-to-study/thanks"
+    template_name = "enquire_to_study/enquire.html"
```

Comment:
**Important**

These templates are fine while we build all the integrations out for the template, However, we aren't going to be able to fulfill the acceptance criteria: 

> All form fields match the designs

The form pages front-end has been done in a previous sprint which is waiting to be merged and deployed to production. Example here https://rca-staging.herokuapp.com/pattern-library/pattern/patterns/pages/forms/form_page--example.html

I think it's worth implementing this as part of this PR due to the acceptance criteria. It will mean our view here should use the templates:

```
    success_url = "patterns/pages/forms/thanks.html"
    template_name = "patterns/pages/forms/form_page.html"
```

It might be worth creating new specific templates in the pattern library for this specific form though, as it's quite custom and may differ from the templates used by the wagtail form builder. In that case, copy those templates so we can work with them:

```
    success_url = "patterns/pages/forms/enquirey_form_thanks.html"
    template_name = "patterns/pages/forms/enquirey_form_page.html"
```

As mentioned, these form templates and styles aren't in master, they are on the `p3-schools` branch, which means we should rebase this off p3-schools. When we are using the correct styles and templates here, we will be able to fulfill the following acceptance criteria:

> All form fields match the designs